lib/ae_mdw/names.ex

```diff
@@ -504,6 +559,39 @@ defmodule AeMdw.Names do
     |> Enum.map(&render_name_info(state, &1, opts))
   end
 
+  defp render_claim(state, plain_name, {{height, _mbi} = block_index, txi}) do
+    tx_hash = Txs.txi_to_hash(state, txi)
+    block_hash = Blocks.block_index_to_hash(state, block_index)
+
+    claim_aetx =
+      case Db.get_tx_data(tx_hash) do
```

Comment:
Looks like an opportunity for reuse creating a function to fetch the tx:
https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/activities.ex#L475-L495
https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/db/name.ex#L173-L200 (here instead comparing the hash would compare the name as well and return the tx)

Reply:
Thanks!